docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -25,11 +83,11 @@
 
 ## Procedure Per IP
 
-This procedure is applied at every experimental insertion (IR1, IR2, IR5, IR8) where local corrections need to be established or checked.
-Keep in mind that this does Beam1 and Beam2 at the same time, but different IPs cannot/ should not be done in parallel.
+This procedure is applied at the main experimental insertions (IR1 and IR5) where local corrections need to be established or checked.
```

Comment:
Didn't we do that in IR2 for ALICE? I thought the local coupling there was the reason we started this?

Reply:
Yes, but for now we only have knobs for IR[15] and Tobias has no plans to do the procedure for IR[28]. Lack of time I presume.